pkgs/development/tools/amazon-qldb-shell/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, clang, cmake, fetchFromGitHub, llvmPackages, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "amazon-qldb-shell";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "awslabs";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Pnm1HxEjjNKpS3tTymtOXxUF7EEnWM+7WBsqeaG8seA=";
+  };
+
+  nativeBuildInputs = [ clang cmake ];
+  buildInputs = [ llvmPackages.libclang ];
+
+  cargoSha256 = "sha256-EUqGSKcGnhrdLn8ystaLkkR31RjEvjW6vRzKPMK77e8=";
+
+  LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
+
+  doInstallCheck = true;
+
+  installCheckPhase = ''
+    $out/bin/qldb --help
+    $out/bin/qldb --version | grep 'qldb ${version}'
+  '';
```

Comment:
You could use [`testVersion`](https://github.com/NixOS/nixpkgs/blob/1b1fe46bb1f2d86cd47353c8d921f6c09677b294/pkgs/build-support/trivial-builders.nix#L785) to do this :)

Reply:
Wasn’t aware of this one, will have a look and update as needed.